src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx

```diff
@@ -0,0 +1,277 @@
+  adapter: netlify(),
+  build: {
+    split: true,
+  },
+});
+```
+
+### 静态网站
+
+对于静态网站，你通常不需要一个适配器。但是，如果你在 Astro 配置中使用 `redirects` 配置，Netlify 适配器可以将其转换为正确的 `_redirects` 格式。
+
+```js
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify/static';
+
+export default defineConfig({
+  adapter: netlify(),
+
+  redirects: {
+    '/blog/old-post': '/blog/new-post',
+  },
+});
+```
+
+只有在你运行 `astro build` 之后，才会有一个 `dist/_redirects` 文件。Netlify 将使用它来正确地路由生产中的页面。
+
+:::note
+你依然可以在 `public/_redirects` 文件中手动添加重定向。你在重定向配置中指定的任何重定向都会被追加到你自己的重定向之后。
+:::
+
+## 用法
+
+[阅读完整的部署指南](/zh-cn/guides/deploy/netlify/)
+
+在 [执行构建](/zh-cn/guides/deploy/#在本地构建站点) 之后，`netlify/` 文件夹将包含 `netlify/functions/` 文件夹中的 [Netlify Functions](https://docs.netlify.com/functions/overview/)。
+
+现在你可以部署了。安装 [Netlify CLI](https://docs.netlify.com/cli/get-started/) 并运行：
+
+```sh
+netlify deploy --build
+```
+
+[Netlify 关于 Astro 的博客文章](https://www.netlify.com/blog/how-to-deploy-astro/) 和 [Netlify 文档](https://docs.netlify.com/integrations/frameworks/astro/) 提供了更多关于如何使用这个集成部署到 Netlify 的信息。
+
+## 配置
+
+为了配置这个适配器，在 `astro.config.mjs` 中的 `netlify()` 函数调用中传递一个对象——只有一个可能的配置选项：
+
+### dist 目录
+
+我们在你的项目根目录下的 `dist` 目录中构建。要更改这个目录，使用 `dist` 选项：
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify/functions';
+
+export default defineConfig({
+  output: 'server',
+  adapter: netlify({
+    dist: new URL('./dist/', import.meta.url),
+  }),
+});
+```
+
+并且在你的 `netlify.toml` 中指向 dist：
+
+```toml title="netlify.toml"
+[functions]
+directory = "dist/functions"
+```
+
+### 构建器
+
+[Netlify 按需构建器](https://docs.netlify.com/configure-builds/on-demand-builders/) 是用于在 Netlify 的 Edge CDN 上构建和缓存页面内容的无服务器函数。你可以使用 `builders` 选项启用这些函数：
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify/functions';
+
+export default defineConfig({
+  output: 'server',
+  adapter: netlify({
+    builders: true,
+  }),
+});
+```
+
+按需构建器仅适用于 `@astrojs/netlify/functions` 适配器，并且与 Edge Functions 不兼容。
+
+### 二进制媒体类型
+
+> 这个选项只适用于 Functions 适配器，不适用于 Edge Functions。
+
+Netlify Functions 需要 `body` 中的二进制数据被 base64 编码。`@astrojs/netlify/functions` 适配器根据 `Content-Type` 头自动处理这个问题。
+我们检查音频、图像和视频文件的常见 mime 类型。要包含应该被视为二进制数据的特定 mime 类型，请使用 `binaryMediaTypes` 选项包含一个二进制 mime 类型列表。
+
+```js {12}
+// src/pages/image.jpg.ts
+
+import fs from 'node:fs';
+
+export function get() {
+  const buffer = fs.readFileSync('../image.jpg');
+
+  // Return the buffer directly, @astrojs/netlify will base64 encode the body
+  return new Response(buffer, {
+    status: 200,
+    headers: {
+      'content-type': 'image/jpeg',
+    },
+  });
+}
+```
+
+## 示例
+
+* [Astro Netlify Edge Starter](https://github.com/sarahetter/astro-netlify-edge-starter) 提供了一个示例以及 README 中的指南。
+
+* [Browse Astro Netlify projects on GitHub](https://github.com/search?q=%22%40astrojs%2Fnetlify%22+filename%3Apackage.json\&type=Code) 提供更多示例!
```

Comment:
@Yan-Thomas Here seems to be an incorrect link.

Reply:
incrrect link, it maybe `https://github.com/search?q=%22%40astrojs%2Fnetlify%22+filename%3Apackage.json&type=repositories`, type should be repositories?

Reply:
@Yan-Thomas how is going on?

Reply:
We have made an PR to update the link, see:
https://github.com/withastro/astro/pull/7862

After it's merged I'll come here, update this and merge!

Reply:
update the link.